### ISSUE TITLE
[META] Fix references to "post" or "posts" when we actually mean any content type, or meant to say "create"

### ISSUE THREAD

**Issue by klonos** [backdrop/backdrop-issues#4568]
This is a follow-up to #3799

While that issue focused on the text shown in the admin UI on the content type configuration form (/admin/structure/types/manage/page), I've found several places where this is happening, both in user-facing text, as well as inline code comments etc.

Let's fix as many of those as possible.

**Comment by klonos** [comment]
...I expect this to take multiple iterations, so making it a meta-issue.

**Comment by klonos** [comment]
I've filed a PR that fixes instances of "post date", such as here:

![image](https://user-images.githubusercontent.com/2423362/90984030-16887080-e57b-11ea-8396-19e3959fd0f7.png)

PS: when going through views code, my eyes hurt from all the coding standards violations for comments, but I refrained from changing them in this PR.

**Comment by stpaultim** [comment]
Is this helpful?

![Test__Comment____PR_3262_backdrop_backdrop_testing_site](https://user-images.githubusercontent.com/3144571/91007760-64fd3580-e5a2-11ea-94e2-417aed520856.png)

**Comment by klonos** [comment]
Thanks @stpaultim 👍 ...I've updated the PR to fix instances of "posted", and some neighbouring instances of "node".

PS: The files the PR was touching increased from 8 to 22, and I would like to keep these changes reviewable. So if there are any more things to be fixed, lets please file separate PRs for them.

**Comment by olafgrabienski** [comment]
> I've updated the PR to fix instances of "posted"

We shouldn't 'fix' all instances of the verb "to post". To "post" a comment is just fine, language-wise. Sure, it's possible to choose other terms like to "add", or to "create", but it doesn't always sound suitable to me.

What do native speakers think?

**Comment by BWPanda** [comment]
I agree with @olafgrabienski in that 'post' as a verb for creating comments is fine. Trying to change that just makes it awkward. Requesting PR be updated to remove changes related to posting comments.
